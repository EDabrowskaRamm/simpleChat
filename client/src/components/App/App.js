import React from 'react'
import { Global } from '@emotion/core'
import { ThemeProvider } from 'emotion-theming'

import globalStyles from '../../styles/global'
import theme from '../../styles/theme/theme'

import { Root } from './App.shards'

function App() {
  return (
    <>
      <Global
        styles={globalStyles}
      />
      <ThemeProvider theme={theme}>
        <Root>
          asd
        </Root>
      </ThemeProvider>
    </>
  )
}

export default App
