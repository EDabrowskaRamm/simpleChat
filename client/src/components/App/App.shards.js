import styled from '@emotion/styled'

export const Root = styled.main`
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: ${({ theme }) => theme.colors.background.primary};
`
