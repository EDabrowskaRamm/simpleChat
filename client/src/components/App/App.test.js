import React from 'react'
import renderer from 'react-test-renderer'
import serializer from 'jest-emotion'

import App from './App'

expect.addSnapshotSerializer(serializer)

describe('renders app', () => {
  it('Snapshot', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
