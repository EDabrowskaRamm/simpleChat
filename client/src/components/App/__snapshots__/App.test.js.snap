// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app Snapshot 1`] = `
.emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #FFF;
}

.emotion-13 {
  width: 100%;
  height: 100vh;
  overflow: auto;
}

@media (min-width:768px) {
  .emotion-13 {
    width: 720px;
    max-height: 1800px;
  }
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 122px;
  background-color: #ebebeb;
  -webkit-letter-spacing: 1.7px;
  -moz-letter-spacing: 1.7px;
  -ms-letter-spacing: 1.7px;
  letter-spacing: 1.7px;
  text-transform: capitalize;
}

.emotion-12 {
  width: 100%;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  height: 120px;
  background-color: #ebebeb;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 50%;
  height: 100%;
  cursor: pointer;
  background-color: #ebebeb;
  border-bottom: 1px solid #e1e3e5;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  background-color: #FFF;
  border-top: 1px solid #e1e3e5;
  border-bottom: 1px solid transparent;
}

.emotion-1:first-of-type {
  border-top-right-radius: 9px;
  border-top-left-radius: 0;
  border-right: 1px solid #e1e3e5;
}

.emotion-1:hover {
  opacity: .9;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 50%;
  height: 100%;
  cursor: pointer;
  background-color: #ebebeb;
  border-bottom: 1px solid #e1e3e5;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  cursor: default;
}

.emotion-2:hover {
  opacity: .9;
}

.emotion-7 {
  display: none;
  display: block;
  width: 100%;
}

.emotion-6 {
  width: 100%;
  padding: 0 30px;
}

.emotion-5 {
  width: 100%;
  padding: 15px 0;
}

.emotion-4 {
  width: 100%;
  padding: 27px 33px;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  border: 2px solid #868c97;
  border-radius: 9px;
}

.emotion-4:placeholder {
  color: #979c9f;
}

.emotion-11 {
  display: none;
}

<main
  className="emotion-14"
>
  <div
    className="emotion-13"
  >
    <header
      className="emotion-0"
    >
      Status meeting standup
    </header>
    <section
      className="emotion-12"
    >
      <div
        className="emotion-3"
      >
        <p
          className="emotion-1"
          disabled={true}
          onClick={[Function]}
        >
          Participants (0)
        </p>
        <p
          className="emotion-2"
          disabled={true}
          onClick={[Function]}
        >
          Chat
        </p>
      </div>
      <div>
        <div
          className="emotion-7"
        >
          <section
            className="emotion-6"
          >
            <div />
            <form
              className="emotion-5"
              onSubmit={[Function]}
            >
              <input
                className="emotion-4"
                id="name"
                onChange={[Function]}
                placeholder="Type in your name"
                type="text"
                value=""
              />
            </form>
          </section>
        </div>
        <div
          className="emotion-11"
        >
          <section
            className="emotion-6"
          >
            <div />
            <form
              className="emotion-5"
              onSubmit={[Function]}
            >
              <input
                className="emotion-4"
                id="message"
                onChange={[Function]}
                placeholder="Message"
                type="text"
                value=""
              />
            </form>
          </section>
        </div>
      </div>
    </section>
  </div>
</main>
`;
